### ISSUE TITLE
[Feature Request]: I made a script that sends an API request to Plex for a partial library update, only for new media.

### ISSUE THREAD

**Issue by samssausages** [tubearchivist/tubearchivist#843]
### This project doesn't accept any new feature requests for the forseeable future. There is no shortage of ideas and the next development steps are clear for years to come.

- [X] I understand that this issue will be closed without comment.

I was having a problem with Plex where I could only get new Tubearchivist media when I did a full Plex Library Update. (Mainly because I'm using a network share that doesn't allow for file system level change notifications)
This was undesirable as I have a very large media library.

So I came up with a way to monitor your library folder for new media, then send an API request to Plex to only do a partial scan, in that specific folder.
Works great in my testing, let me know if you have issues.
Hope it helps someone!
Feel free to add to your userscripts section, if you need me to do a PR, I can.

https://github.com/samssausages/plex_scripts/tree/main

**Comment by samssausages** [comment]
I was having a problem with Plex where I could only get new Tubearchivist media when I did a full Plex Library Update. (Mainly because I'm using a network share that doesn't allow for file system level change notifications)
So I came up with a way to monitor your library folder for new media, then send an API request to Plex to only do a partial scan, in that specific folder.
Works great in my testing, let me know if you have issues.
Hope it helps someone!
https://github.com/samssausages/plex_scripts/tree/main
